src/main/java/code401challenges/MultiBracketValidation.java

```diff
@@ -0,0 +1,40 @@
+package code401challenges;
+
+import org.apache.commons.math3.exception.NullArgumentException;
+
+import java.util.Stack;
+
+public class MultiBracketValidation {
+    public static boolean multiBracketValidation(String s) {
+        //set a placeholder for the current string
+        String curChar = null;
+        if (s == null) {
+            throw new NullArgumentException();
+        }
+//       create stacks for
+        Stack<String> lBrackets = new Stack<>();
+
+// ********************* how could i do this recursively?****************************

```

Comment:
I think this is a problem that will be easier to solve iteratively rather than with recursion.